src/strings/unicode_ops.c

```diff
@@ -1,3 +1,56 @@
+/* Compares two strings, returning -1, 0 or 1 to indicate less than,
+ * equal or greater than based on Unicode Collation Algorithm */
+MVMint64 MVM_unicode_string_compare(MVMThreadContext *tc, MVMString *a, MVMString *b) {
+    MVMStringIndex alen, blen, i, scanlen;
+    // Interation variables
```

Comment:
`Int`eration is a good pun given they're integers, but maybe not quite what was meant :-D